### PR TITLE
Fix error response to be valid JSON.

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: Sources/ITwinMobile/ITMMessenger.swift [hunk 1]
```diff
@@ -21,6 +21,19 @@ internal extension JSONSerialization {
         return string(withITMJSONObject: object, prettyPrint: false)
     }
 
+    /// Determines if value is a serializable type, i.e., one that can be passed into JSONSerialization.data(withJSONObject:)
+    /// - Parameter value: value to check
+    /// - Returns: true if value is a Dictionary or Array, false otherwise.
+    static func isSerializableType(_ value: Any) -> Bool {
```
Comment (toddsouthenbentley) [inline Sources/ITwinMobile/ITMMessenger.swift:27]: From the description of this function is seems like it is trying to accomplish the same thing as JSONSerialization.isValidJSONObject. If it isn't the same then I think some clarification of the documentation is needed here.

Comment (tcobbs-bentley) [reply Sources/ITwinMobile/ITMMessenger.swift:27]: I was unaware of that function. I switch to it and test to verify it works.

Comment (tcobbs-bentley) [reply Sources/ITwinMobile/ITMMessenger.swift]: Fixed.

File: Sources/ITwinMobile/ITMMessenger.swift [hunk 2]
```diff
@@ -29,11 +29,29 @@ internal extension JSONSerialization {
...
         }
+        let wrapped: Bool
+        let validJSONObject: Any
+        if JSONSerialization.isValidJSONObject(object) {
+            wrapped = false
+            validJSONObject = object
+        } else {
+            wrapped = true
+            // Wrap object in an array
+            validJSONObject = [object]
```
Comment (toddsouthenbentley) [inline Sources/ITwinMobile/ITMMessenger.swift:40]: Will wrapping an invalid object in an array always make it valid for JSONSerialization? If not, then we should call
```swift
JSONSerialization.isValidJSONObject(validJSONObject)
```

Comment (tcobbs-bentley) [reply Sources/ITwinMobile/ITMMessenger.swift:40]: This is internal, and as far as I know, it will always work in times that it is called.

**Review by toddsouthenbentley — APPROVED** [review]

**Pull request merged**